main.tf

```
@@ -7,24 +7,32 @@ data "aws_iam_policy" "cloudwatch_readonly" {
 }
 
 module "lambda" {
-  source = "git::https://github.com/plus3it/terraform-aws-lambda.git?ref=v1.3.0"
-
-  function_name = var.name
-  description   = "Post messages from AWS to Slack"
-  handler       = "aws-to-slack/src/index.handler"
-  runtime       = "nodejs14.x"
-  timeout       = 10
+  source = "git::https://github.com/terraform-aws-modules/terraform-aws-lambda?ref=v5.0.0"
+
+  function_name            = var.name
+  description              = "Post messages from AWS to Slack"
+  handler                  = "aws-to-slack/src/index.handler"
+  artifacts_dir            = var.lambda.artifacts_dir
+  build_in_docker          = var.lambda.build_in_docker
+  create_package           = var.lambda.create_package
+  ephemeral_storage_size   = var.lambda.ephemeral_storage_size
+  ignore_source_code_hash  = var.lambda.ignore_source_code_hash
+  local_existing_package   = var.lambda.local_existing_package
+  memory_size              = var.lambda.memory_size
+  recreate_missing_package = var.lambda.recreate_missing_package
+  runtime                  = var.lambda.runtime
+  s3_bucket                = var.lambda.s3_bucket
+  s3_existing_package      = var.lambda.s3_existing_package
+  s3_prefix                = var.lambda.s3_prefix
+  store_on_s3              = var.lambda.store_on_s3
+  timeout                  = var.lambda.timeout
 
   source_path = "${path.module}/vendor"
```

The `source_path` also needs to be updated to match the new approach...

```suggestion
  source_path = [
    {
      path             = "${path.module}/src"
      pip_requirements = true
      patterns         = ["!\\.terragrunt-source-manifest"]
    }
  ]
```